contents/blog/learnings-from-elon.md

```diff
@@ -0,0 +1,95 @@
+
+## He’s erratic
+
+Elon has had a string of weird stuff on social media. Whilst this probably brings more awareness and a strange kind of marketing benefit, we think it’s a symptom of a very top down culture where things change based on the leader’s mood.
+
+We do not think this is a strength - perhaps it works in his companies because it creates a fear element, and more urgency as a result. 
+
+At PostHog, as mentioned, we’re more intrinsically motivated so ruling by fear would be harmful to performance.
+
+Whilst Tim and I technically are the CTO and CEO respectively, we more or less share the CEO role. We’ve been very effective like this. I think it’s because we’re a product led company, so what we ship has a very strong impact on marketing and sales (see “Product drives marketing and sales” above), so keeping a tight connection between these varied teams really matters.
+
+The key to making a split like this work is happily spending a lot of time together. This (and being ok sometimes missing out on decisions) is the only way to be decisive when two of you are paired like this. We talk a lot (several hours a week, minimum) - which can feel inefficient, but the reality is we need to optimize for organizational effectiveness, not our own schedules.
+
+## Likes the taste of his own blood
+
+Clickbait subtitle. You got me. Elon gives very direct feedback and makes painful decisions rapidly and doesn’t dwell on any of this stuff. For example, he 
+
+We describe PostHog as a team for this reason. We have performance managed - letting people go decisively - throughout our time and it has been one of the most important things we’ve ever done.
+
+We do this because we’re optimizing for our strongest performers, who want to work in an environment where everyone is shipping their best work. As soon as we make it ok not to push hard, everyone will feel it and eventually quit or reduce their own input.
+
+We try to balance this by being as kind as possible for people who depart under these circumstances - we pay four months’ severance once someone has been here more than a couple of months. Beyond pay, we talk directly and honesty to people - we avoid doing PIP (Performance Improvement Program) type processes as we think they are dehumanizing on both sides and cause more legal cases than they solve.
+
+## Fun is a serious business
+
+There’s very little written in startup lore about the importance of having fun. Elon seems to get a clear kick of out what he does.
+
+We often make decisions at PostHog based on what will be the most entertaining. This can lead to remarkable things happening with our website, marketing, customer success and product. The reason we do this is that we want to build all the products in one. That will take a long time as it’s a lot of products, so we need to consider PostHog a marathon, and fun keeps people’s spirits up.
```

Comment:
```suggestion
We often make decisions at PostHog based on what will be the most entertaining. This can lead to remarkable things happening with our website, marketing, customer success and product. The reason we do this is that we have a lot of products to build which will take a long time. We need to consider PostHog a marathon, and fun keeps people’s spirits up.
```